pkg/graph/resolvers/system_intake_status_admin_test.go

```diff
@@ -0,0 +1,387 @@
+			intake: models.SystemIntake{
+				Step:          models.SystemIntakeStepDECISION,
+				DecisionState: models.SIDSLcidIssued,
+				State:         models.SystemIntakeStateOPEN,
+			},
+			expectedStatus: models.SISALcidIssued,
+			expectError:    false,
+		},
+		{
+			testCase: "Decision LCID issued, closed",
+			intake: models.SystemIntake{
+				Step:          models.SystemIntakeStepDECISION,
+				DecisionState: models.SIDSLcidIssued,
+				State:         models.SystemIntakeStateCLOSED,
+			},
+			expectedStatus: models.SISALcidIssued,
+			expectError:    false,
+		},
+		{
+			testCase: "Decision No Governance, open",
+			intake: models.SystemIntake{
+				Step:          models.SystemIntakeStepDECISION,
+				DecisionState: models.SIDSNoGovernance,
+				State:         models.SystemIntakeStateOPEN,
+			},
+			expectedStatus: models.SISANoGovernance,
+			expectError:    false,
+		},
+		{
+			testCase: "Decision No Governance, closed",
+			intake: models.SystemIntake{
+				Step:          models.SystemIntakeStepDECISION,
+				DecisionState: models.SIDSNoGovernance,
+				State:         models.SystemIntakeStateCLOSED,
+			},
+			expectedStatus: models.SISANoGovernance,
+			expectError:    false,
+		},
+		{
+			testCase: "Decision Not Approved, open",
+			intake: models.SystemIntake{
+				Step:          models.SystemIntakeStepDECISION,
+				DecisionState: models.SIDSNotApproved,
+				State:         models.SystemIntakeStateOPEN,
+			},
+			expectedStatus: models.SISANotApproved,
+			expectError:    false,
+		},
+		{
+			testCase: "Decision Not Approved, closed",
+			intake: models.SystemIntake{
+				Step:          models.SystemIntakeStepDECISION,
+				DecisionState: models.SIDSNotApproved,
+				State:         models.SystemIntakeStateCLOSED,
+			},
+			expectedStatus: models.SISANotApproved,
+			expectError:    false,
+		},
+		// These cases should not be present in the actual app, but they are possible edge cases
+		{
+			testCase: "Decision No Decision, closed",
+			intake: models.SystemIntake{
+				Step:          models.SystemIntakeStepDECISION,
+				DecisionState: models.SIDSNoDecision,
+				State:         models.SystemIntakeStateCLOSED,
+			},
+			expectedStatus: models.SISAClosed,
+			expectError:    false,
+		},
+		// This case will be handled in application code to ensure this doesn't occur. If needed, further logic can be created to enforce it, such as a check constraint in the DB
+		{
+			testCase: "Decision No Decision, closed",
+			intake: models.SystemIntake{
+				Step:          models.SystemIntakeStepDECISION,
+				DecisionState: models.SIDSNoDecision,
+				State:         models.SystemIntakeStateOPEN,
+			},
+			expectedStatus: models.SISAInitialRequestFormInProgress,
+			expectError:    true,
+		},
+	}
+	systemIntakeAdminStatusRunTestCollection(t, intakeFormTests, "Testing all system intake form Statuses")
+	systemIntakeAdminStatusRunTestCollection(t, draftBizCaseTests, "Testing all draft Buisness Case Statuses")
```

Comment:
Typo here (and when running `finalBizCaseTests`) - current text says "Buisness" instead of "Business". I think this also shows up in the `pkg/graph/resolvers/system_intake_status_admin.go` code.

Reply:
VS Code has a spellchecker extension; I highly recommend it.

Reply:
Thank you both, I'll update the instances of it. @mynar7, I just tried the extension, but we probably need a pass to fix other spelling error instances before it's useful. It makes too many problem alerts for acronyms etc.